src/_Problems_/reverse-number/index.js

```diff
@@ -15,6 +15,47 @@ function reverseNumber(num) {
   return reverse * Math.sign(num);
 }
 
+/**
+ * 
+ * Given a 32-bit signed integer, reverse digits of an integer.
+
+  Example 1:
+
+  Input: 123
+  Output: 321
+  Example 2:
+
+  Input: -123
+  Output: -321
+  Example 3:
+
+  Input: 1534236469
+  Output: 0 // overflows
+  Note:
+  Assume we are dealing with an environment which could only 
+  store integers within the 32-bit signed integer range: [−2^31,  2^31 − 1]. 
```

Comment:
Try to solve problem for input > 2^31 -1 .
It is also possible.

Reply:
Yes, it can be done, I tried that before adding the `return 0;` block. I simply posted the solution for one of the coding challenge platforms.